src/together/resources/finetune.py

```diff
@@ -366,17 +380,78 @@ def list_events(self, id: str) -> FinetuneListEvents:
             ),
             stream=False,
         )
-
         assert isinstance(response, TogetherResponse)
 
         return FinetuneListEvents(**response.data)
 
+    def list_checkpoints(self, id: str) -> List[FinetuneCheckpoint]:
+        """
+        List available checkpoints for a fine-tuning job
+
+        Args:
+            id (str): Unique identifier of the fine-tune job to list checkpoints for
+
+        Returns:
+            List[FinetuneCheckpoint]: List of available checkpoints
+        """
+        events = self.list_events(id).data or []
+
+        checkpoints: List[FinetuneCheckpoint] = []
+
+        for event in events:
+            event_type = event.type
+
+            if event_type == FinetuneEventType.CHECKPOINT_SAVE:
+                formatted_time = format_event_timestamp(event)
+                step = get_event_step(event)
+                checkpoint_name = f"{id}:{step}" if step else id
+
+                checkpoints.append(
+                    FinetuneCheckpoint(
+                        type="Intermediate",
```

Comment:
What do you think about listing the step here as well, like `"Intermediate (step 20)"`? It's not completely obvious how to interpret the table now, but if we add a bit of extra information, it will be clearer to the user what the names stand for